index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/06aa7828ae.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+<!-- <header class="toolbar">
+  <a href="#" class="toolbar logo"><img src="images/logo.png" alt=""></a>
+  <nav class="toolbar menu"><img src="images/menu.png" alt=""></nav>
+</header>
+<section class ="container">
+  <div class="bg-image1"></div>
+  <div class="bg-image2"></div>
+  <div class="bg-image3"></div>
+  <div class="bg-image4"></div>
+  <div class="bg-image6"></div>
+  <div class="bg-image7"></div>
+  <div class="bg-imagea"></div>
+  <div class="bg-imageb"></div> -->
+     <!-- <div class="intro"><h1>I'm Victor,
+       Glad to see you!</h1></div>
+          <div class="content">
+           <p>Hello! I am a software developer! I can help you build a product, feature or website.
+           Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
+          </div>
+          
+     <ul class="social">
+       <li><img src="images/connect.png" alt=""></li>
+     </ul>
+     <ul>
```

Comment:
- [ ] [OPTIONAL] kindly remove unused code from your codebase to make your code clean and readable.